tests/platform_test_harness/graphics_platform_test_harness.cpp

```diff
@@ -611,37 +611,34 @@ int main(int argc, char const** argv)
     bool success = true;
     try
     {
-        success &= test_probe(platform_dso, config);
-        if (success)
+        test_probe(platform_dso, config);
```

Comment:
I think this should still be `success &= test_probe()`; failure to return a supported probe value will result in the platform failing to work.

Reply:
Done